web/src/js/__tests__/components/Header/__snapshots__/CaptureMenuSpec.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CaptureMenu Component should render correctly 1`] = `null`;
```

Comment:
Surprised there is nothing rendered here?

Reply:
@mhils probably that's why: 
```
export default function CaptureMenu() {
    //empty component since we don't have an header menu for capture, but we just display the title
    return <></>;
}
```

Reply:
Addressed in #7000. :)